### PR TITLE
updates for Play 2.7.0-RC9

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: src/main/resources/2.7.x.conf [hunk 1]
```diff
@@ -118,7 +121,7 @@ templatecontrol {
 
           "com.h2database" = """    play "com.h2database:h2:1.4.197""""
 
-          "scalatestplus-play" = """    playTest "org.scalatestplus.play:scalatestplus-play_\$scalaVersion:4.0.0-RC2""""
+          "scalatestplus-play" = """    playTest "org.scalatestplus.play:scalatestplus-play_$scalaVersion:4.0.0-RC2""""
```
Comment (dwijnand) [inline src/main/resources/2.7.x.conf]: Is this right?

File: src/main/resources/2.7.x.conf [hunk 2]
```diff
@@ -98,7 +99,9 @@ templatecontrol {
         conversions = {
 
           "def scalaVersion" = """def scalaVersion = System.getProperty("scala.binary.version", /* default = */ "2.12")"""
-          "def playVersion" = """def playVersion = "2.7.0-RC8""""
+          "def playVersion" = """def playVersion = "2.7.0-RC9""""
+
+          """play "org.webjars:webjars-play""" = """    play "org.webjars:webjars-play_$scalaVersion:2.7.0-RC9"""
```
Comment (dwijnand) [inline src/main/resources/2.7.x.conf]: Should this be:
```suggestion
          """play "org.webjars:webjars-play""" = """    play "org.webjars:webjars-play_\$scalaVersion:2.7.0-RC9"""
```
?

File: src/main/resources/2.7.x.conf (hunk 1 above)
Comment (renatocaval) [reply src/main/resources/2.7.x.conf]: yes, this is right for now. We have a few samples using scalatest plus. They will all get broken if we escape the $. 

When we don't escape the $, we will only break play-scala-seed.g8. So, better to break only one and fix it by hand. 

And a fix is coming, see #102

File: src/main/resources/2.7.x.conf (hunk 2 above)
Comment (renatocaval) [reply src/main/resources/2.7.x.conf]: no, for the same reasons mentioned earlier.

We should only escape for Giter8 templates.

File: src/main/resources/2.7.x.conf (hunk 1 above)
Comment (dwijnand) [reply src/main/resources/2.7.x.conf]: I see.  Thanks for summarising the state!

**Pull request merged**